### ISSUE TITLE
Hermes debugger no longer working after 0.126.0

### ISSUE THREAD

**Issue by nibblesnbits** [facebook/flipper#3183]
## 🐛 Bug Report
I was able to use the Hermes Debugger for React Native yesterday with version 0.125.0, but now that I've installed 0.126.0, the debugger is listed under "Unavailable Plugins" and when selected I get the message

```
Plugin 'Hermes Debugger (RN)' is unavailable
Device plugin 'Hermes Debugger (RN)' is not supported by the selected device 'iPhone' (iOS).
```

## To Reproduce
Run any iOS app, and select the app in Flipper.

## Environment
iOS 15.1
React Native 0.66.4
@react-native-community/cli-debugger-ui 5.0.1
(Note that none of the above have changed since yesterday, and everything worked then).
Flipper 0.126.0

**Comment by mweststrate** [comment]
Hermes Debugger RN is not supported by iOS devices, it is supported by Metro / React Native. Make sure you have Metro running to see the device.

**Comment by hakkikonu** [comment]
agree. 0.126 crashes metro bundler also. i downgraded to 0.125 and now it working properly.

**Comment by mweststrate** [comment]
Sorry, missed that this is most probably a duplicate of #3189, but since downgrading fixes the problem, pretty sure it is. Upgrading to 0.127 that contains a fix should work (just released), or otherwise this work around on 0.126 should work as well: https://github.com/facebook/flipper/issues/3189#issuecomment-997191720.

**Comment by hakkikonu** [comment]
Thanks
